main.go

```diff
@@ -332,15 +534,15 @@ func (c MainController) LoginHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func (c MainController) ForbiddenHandler(w http.ResponseWriter, r *http.Request) {
-	renderPage(w, "403", PageData{PageName: "403 - Forbidden", IsAdmin: getUserFromContext(r.Context()).Admin})
+	renderPage(w, "403", PageData{PageName: "403 - Forbidden", MainRole: getUserMainRole(getUserFromContext(r.Context()))})
```

Comment:
I'd change renderPage to fill in MainRole so we don't have to duplicate this logic everywhere.

Reply:
👍